preload/cls/IPM/Installer.cls

```diff
@@ -35,6 +35,7 @@ XData PM [ XMLNamespace = INSTALLER ]
     <Arg Value="${MODDIR}"/>
   </Invoke>
   <Invoke Class="IPM.Installer" Method="ZPMCompile" CheckStatus="true" />
+  <Invoke Class="IPM.Installer" Method="MapIfLegacy" CheckStatus="true" />
```

Comment:
My understanding is that `<Invoke>`s in module.xml should be a (not necessarily proper) subset of Installer.cls since a legacy user can always download a release artifact from GitHub or community PM registry. 

Not sure if there are `<Invoke>`s that we only want present in Installer.cls but not module.xml.

@isc-tleavitt Thoughts?

Reply:
Wait - I'm wrong, we shouldn't need this because the above (line 34) call to ZPMLoad will end up running all the `<Invoke>`s.

Reply:
I was confusing this with:
`<Invoke Class="IPM.Installer" Method="Map" Phase="Reload" When="Before" />`

Reply:
@isc-shuliu @isc-tleavitt so should the installer invokes be removed in favor of the Invoke's in module.xml? The installer then essentially installs the module using itself which would run invokes from the module.xml?